### ISSUE TITLE
[Bug] exported as a JSON file in the Windows installation version appears as unreadable content.（windows安装版的对话导出的JSON为乱码。）

### ISSUE THREAD

**Issue by modderBUG** [Yidadaa/ChatGPT-Next-Web#3395]
**Describe the bug**
The conversation exported as a JSON file in the Windows installation version appears as garbled or unreadable content.
windows安装版的对话导出的JSON文件为乱码

**To Reproduce**
1. Go to 'Setting' 打开设置
2. Click on 'Export' 点击对话导出
3. Open the JSON File 打开JSON文件

**Expected behavior**
The expected behavior is to have readable JSON content instead of garbled characters.
希望是可阅读的JSON内容，而不是乱码。

**Screenshots**
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/36789418/6140b3dd-3e6e-4920-a7e8-93c88c2e52d6)
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/36789418/9eb5f184-ee7d-4087-b82f-a6854fa541ad)

**Comment by H0llyW00dzZ** [comment]
can you screenshot via here ?
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/a221d5de-eebd-44de-9344-78cae8e75625)

it's works fine for me nothing issue with unicode/character chinese

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/f12dfecd-098c-4450-bddb-5acd08d64481)

**Comment by modderBUG** [comment]
> can you screenshot via here ?
> 
> ![image](https://private-user-images.githubusercontent.com/17626300/286095056-a221d5de-eebd-44de-9344-78cae8e75625.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExNDE1MzMsIm5iZiI6MTcwMTE0MTIzMywicGF0aCI6Ii8xNzYyNjMwMC8yODYwOTUwNTYtYTIyMWQ1ZGUtZWViZC00NGRlLTkzNDQtNzhjYWU4ZTc1NjI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDAzMTM1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU3YjUyZTlkODM2NDk1ODg0MGY5MDAyN2U1OGM3MGI4NjdmMmZlOTg4Y2JmYjcyNGU5ZTBiZDM3ZGNlMGQ1NzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wqCmz9zafOhH3DHRl4h57cN6LlNPR4tzL8pyShydQiw)
> 
> it's works fine for me nothing issue with unicode/character chinese
> 
> ![image](https://private-user-images.githubusercontent.com/17626300/286094999-f12dfecd-098c-4450-bddb-5acd08d64481.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExNDE1MzMsIm5iZiI6MTcwMTE0MTIzMywicGF0aCI6Ii8xNzYyNjMwMC8yODYwOTQ5OTktZjEyZGZlY2QtMDk4Yy00NDUwLWJkZGItNWFjZDA4ZDY0NDgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDAzMTM1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUyNTVmNjE2OGJhODg5YWIyZmRiZDRhMjFjYzBhOWQwYmFiNmUyZjY5NGFlMTNhNTQxZGY0ZGFiMTRhNWVjZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.s8elmBx0JSnYuhp46LopciRYfFJQuvayO9MSuM-8Z84)

yes 
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/36789418/ad7f85ec-7f60-408d-b1fa-1c55a495750e)
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/36789418/0c7de275-84cc-4e33-8258-01829e36e71a)

**Comment by modderBUG** [comment]
Before exporting, it appears normal, as shown in the above image. However, after the export, it becomes as follows:

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/36789418/001d54d3-b687-4ffc-89fd-3f09283dc1bc)

**Comment by H0llyW00dzZ** [comment]
> Before exporting, it appears normal, as shown in the above image. However, after the export, it becomes as follows:
> 
> ![image](https://private-user-images.githubusercontent.com/36789418/286105907-001d54d3-b687-4ffc-89fd-3f09283dc1bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExNDI1NTgsIm5iZiI6MTcwMTE0MjI1OCwicGF0aCI6Ii8zNjc4OTQxOC8yODYxMDU5MDctMDAxZDU0ZDMtYjY4Ny00ZmZjLTg5ZmQtM2YwOTI4M2RjMWJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDAzMzA1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4ZTc2MDU4MTAxM2ZkNjkxNjEzZjYzZGFmZTg1ODk4NjMzZmQ4MGJhYTA0YjNkYzRiZTI4MjdhYTRkZjZlNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4hNsu_X5TwEzdPGc5TDILXE831WCu7plAOXZGsHQi90)

its because of this I am pretty sure

https://github.com/Yidadaa/ChatGPT-Next-Web/blob/2c12be62c4574f7914ff4075560b52298fbe64c7/app/utils.ts#L34C1-L78C2

code :
```ts
export async function downloadAs(text: string, filename: string) {
  if (window.__TAURI__) {
    const result = await window.__TAURI__.dialog.save({
      defaultPath: `${filename}`,
      filters: [
        {
          name: `${filename.split('.').pop()} files`,
          extensions: [`${filename.split('.').pop()}`],
        },
        {
          name: "All Files",
          extensions: ["*"],
        },
      ],
    });

    if (result !== null) {
      try {
        await window.__TAURI__.fs.writeBinaryFile(
          result,
          new Uint8Array([...text].map((c) => c.charCodeAt(0)))
        );
        showToast(Locale.Download.Success);
      } catch (error) {
        showToast(Locale.Download.Failed);
      }
    } else {
      showToast(Locale.Download.Failed);
    }
  } else {
    const element = document.createElement("a");
    element.setAttribute(
      "href",
      "data:text/plain;charset=utf-8," + encodeURIComponent(text),
    );
  element.setAttribute("download", filename);

  element.style.display = "none";
  document.body.appendChild(element);

  element.click();

  document.body.removeChild(element);
}
}
```

The default function `downloadAs` must output JSON to enable its functionality.

example:

```ts
//To ensure the expected functionality, the default file format must be JSON.
export async function downloadAs(text: object, filename: string) {
  const json = JSON.stringify(text);
  const blob = new Blob([json], { type: "application/json" });
  const arrayBuffer = await blob.arrayBuffer();
  const uint8Array = new Uint8Array(arrayBuffer);

  try {
    if (window.__TAURI__) {
      /**
       * Fixed client app [Tauri]
       * Resolved the issue where files couldn't be saved when there was a `:` in the dialog.
      **/
      const fileName = filename.replace(/:/g, '');
      const fileExtension = fileName.split('.').pop();
      const result = await window.__TAURI__.dialog.save({
        defaultPath: `${fileName}`,
        filters: [
          {
            name: `${fileExtension} files`,
            extensions: [`${fileExtension}`],
          },
          {
            name: "All Files",
            extensions: ["*"],
          },
        ],
      });

      if (result !== null) {
        await window.__TAURI__.fs.writeBinaryFile(
          result,
          Uint8Array.from(uint8Array)
        );
        showToast(Locale.Download.Success);
      } else {
        showToast(Locale.Download.Failed);
      }
    } else {
      const url = URL.createObjectURL(blob);
      const anchor = document.createElement("a");
      anchor.href = url;
      anchor.download = `${filename}`;
      anchor.click();
      URL.revokeObjectURL(url);
      showToast(Locale.Download.Success);
    }
  } catch (error) {
    showToast(Locale.Download.Failed);
  }
}
```

then other need to refactor in `store/sync.ts`

https://github.com/Yidadaa/ChatGPT-Next-Web/blob/main/app/store/sync.ts#L60C1-L68C7
code
```ts
    export() {
      const state = getLocalAppState();
      const datePart = isApp
      ? `${new Date().toLocaleDateString().replace(/\//g, '_')} ${new Date().toLocaleTimeString().replace(/:/g, '_')}`
      : new Date().toLocaleString();

      const fileName = `Backup-${datePart}.json`;
      downloadAs(JSON.stringify(state), fileName);
    },
```

replace to this:
```ts
    export() {
      const state = getLocalAppState();
      const datePart = isApp
        ? `${new Date().toLocaleDateString().replace(/\//g, '_')} ${new Date().toLocaleTimeString().replace(/:/g, '_')}`
        : new Date().toLocaleString();

      const fileName = `Backup-${datePart}.json`;
      downloadAs((state), fileName);
    },
```

**Comment by H0llyW00dzZ** [comment]
> [!NOTE]  
> the code I provide it work's fine for me without any getting issue while exporting it as manually in my website https://chatgpt.btz.sh/ and my desktop app

**Comment by H0llyW00dzZ** [comment]
ref (I used) : https://developer.mozilla.org/en-US/docs/Web/API/Blob

**Comment by H0llyW00dzZ** [comment]
proof no issue and its correct syntax of json in unicode/char chinese

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/29d585ba-d94c-4e56-9a45-abe9a3d7fd28)

**Comment by modderBUG** [comment]
> ref (I used) : https://developer.mozilla.org/en-US/docs/Web/API/Blob

Thank you. Could you directly compile an executable (exe) for me to try? I'm unable to install the Yarn environment.

**Comment by H0llyW00dzZ** [comment]
> > ref (I used) : https://developer.mozilla.org/en-US/docs/Web/API/Blob
> 
> Thank you. Could you directly compile an executable (exe) for me to try? I'm unable to install the Yarn environment.

yes

**Comment by H0llyW00dzZ** [comment]
@modderBUG

Link Download: https://efss.qloud.my/index.php/s/mxmKK4ayJbdWQNA

**Comment by jkjoker** [comment]
> proof no issue and its correct syntax of json in unicode/char chinese
> 
> ![image](https://private-user-images.githubusercontent.com/17626300/286124331-29d585ba-d94c-4e56-9a45-abe9a3d7fd28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExODYwMjgsIm5iZiI6MTcwMTE4NTcyOCwicGF0aCI6Ii8xNzYyNjMwMC8yODYxMjQzMzEtMjlkNTg1YmEtZDk0Yy00ZTU2LTlhNDUtYWJlOWEzZDdmZDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDE1MzUyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3OWNiZGZjMWI4OGJmZmJjNjU3NmM4YmRhNjBhODViNGY0YWEzMjdlOTFkMzg2OGVlNGZhYjE4MjI1ZWMzZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Q-LKhglMR2Y2o_nx1mlTSSuar2Yacwnd5C86ItfyK-c)

I'm glad to see that you have found the reason. The effect shown in this picture is what I want to see. I want to export the JSON file and open it in VS code to see this effect. I hope this issue can be fixed in the new version, thank you.

**Comment by H0llyW00dzZ** [comment]
> > proof no issue and its correct syntax of json in unicode/char chinese
> > ![image](https://private-user-images.githubusercontent.com/17626300/286124331-29d585ba-d94c-4e56-9a45-abe9a3d7fd28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExODYwMjgsIm5iZiI6MTcwMTE4NTcyOCwicGF0aCI6Ii8xNzYyNjMwMC8yODYxMjQzMzEtMjlkNTg1YmEtZDk0Yy00ZTU2LTlhNDUtYWJlOWEzZDdmZDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDE1MzUyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3OWNiZGZjMWI4OGJmZmJjNjU3NmM4YmRhNjBhODViNGY0YWEzMjdlOTFkMzg2OGVlNGZhYjE4MjI1ZWMzZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Q-LKhglMR2Y2o_nx1mlTSSuar2Yacwnd5C86ItfyK-c)
> 
> I'm glad to see that you have found the reason. The effect shown in this picture is what I want to see. I want to export the JSON file and open it in VS code to see this effect. I hope this issue can be fixed in the new version, thank you.

in VS code it's corrected as well (even its in minify mode)

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/6aec9869-788b-4004-b0ae-1f59b8278141)

if we unminify

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/a9799ce2-9ffe-486c-b1ef-afcaedf91692)

**Comment by H0llyW00dzZ** [comment]
> [!NOTE]  
> Also additional note, for viewing lots of data recommended using tool, which can handling json, because if viewing manually such as in VS code editor or other, it might causing lag/not corrected while rendering (in fact its already corrected json).

**Comment by jkjoker** [comment]
> > > proof no issue and its correct syntax of json in unicode/char chinese
> > > ![image](https://private-user-images.githubusercontent.com/17626300/286124331-29d585ba-d94c-4e56-9a45-abe9a3d7fd28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExODYwMjgsIm5iZiI6MTcwMTE4NTcyOCwicGF0aCI6Ii8xNzYyNjMwMC8yODYxMjQzMzEtMjlkNTg1YmEtZDk0Yy00ZTU2LTlhNDUtYWJlOWEzZDdmZDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDE1MzUyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3OWNiZGZjMWI4OGJmZmJjNjU3NmM4YmRhNjBhODViNGY0YWEzMjdlOTFkMzg2OGVlNGZhYjE4MjI1ZWMzZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Q-LKhglMR2Y2o_nx1mlTSSuar2Yacwnd5C86ItfyK-c)
> > 
> > 
> > I'm glad to see that you have found the reason. The effect shown in this picture is what I want to see. I want to export the JSON file and open it in VS code to see this effect. I hope this issue can be fixed in the new version, thank you.
> 
> in VS code it's corrected as well (even its in minify mode)
> 
> ![image](https://private-user-images.githubusercontent.com/17626300/286333402-6aec9869-788b-4004-b0ae-1f59b8278141.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEyNDQ2NjEsIm5iZiI6MTcwMTI0NDM2MSwicGF0aCI6Ii8xNzYyNjMwMC8yODYzMzM0MDItNmFlYzk4NjktNzg4Yi00MDA0LWIwYWUtMWY1OWI4Mjc4MTQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI5VDA3NTI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhZjc4MGI4NDk0N2ViZDRjM2Q2ZTJkZDYyYWY4NTRmY2I1OGMzNDgzY2UwNWFiZTFhNmIxY2RhZjk0MTA1MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.05CsZGPlt28Qoo9pwg4RghP4mcgQ6mSFy4GLGEDKM3s)
> 
> if we unminify
> 
> ![image](https://private-user-images.githubusercontent.com/17626300/286333713-a9799ce2-9ffe-486c-b1ef-afcaedf91692.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEyNDQ2NjEsIm5iZiI6MTcwMTI0NDM2MSwicGF0aCI6Ii8xNzYyNjMwMC8yODYzMzM3MTMtYTk3OTljZTItOWZmZS00ODZjLWIxZWYtYWZjYWVkZjkxNjkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI5VDA3NTI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1ZGE0OWQyYWQ0OTYxNWM0MmVkODIyZTAyMTk3ZDhlNzZhOTZmYzViNWM3ZDBlNjg4YWE4MTI5NjY4YmM0ZDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O3zi4AMALhOYIHZa0RMXxTunSArky_-cvCf4fmFLY-o)

找到问题了：
https://github.com/Yidadaa/ChatGPT-Next-Web/issues/3394#issuecomment-1831383092

**Comment by modderBUG** [comment]
I tested the export and import functions using your executable, and everything is working fine now.

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/36789418/37c586eb-c776-4fb6-8431-114eb4e30f0e)

**Comment by H0llyW00dzZ** [comment]
> I tested the export and import functions using your executable, and everything is working fine now.
> 
> ![image](https://private-user-images.githubusercontent.com/36789418/286792026-37c586eb-c776-4fb6-8431-114eb4e30f0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMTA2ODYsIm5iZiI6MTcwMTMxMDM4NiwicGF0aCI6Ii8zNjc4OTQxOC8yODY3OTIwMjYtMzdjNTg2ZWItYzc3Ni00ZmI2LTg0MzEtMTE0ZWI0ZTMwZjBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDAyMTMwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNTkxOGZiOTk1OGI4ZDgwMWQxMDI4YmYzMWQ0NzM1Y2EwYTlhNTZkZjNiNmM2NjM1YWJmZjRlMGVlYjYwYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M9iFax2VcHuQdylzP7Owl5f7QGiq_mOnJUdk2bHAGDs)

nice

**Comment by xingty** [comment]
> > I tested the export and import functions using your executable, and everything is working fine now.
> > ![image](https://private-user-images.githubusercontent.com/36789418/286792026-37c586eb-c776-4fb6-8431-114eb4e30f0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMTA2ODYsIm5iZiI6MTcwMTMxMDM4NiwicGF0aCI6Ii8zNjc4OTQxOC8yODY3OTIwMjYtMzdjNTg2ZWItYzc3Ni00ZmI2LTg0MzEtMTE0ZWI0ZTMwZjBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDAyMTMwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNTkxOGZiOTk1OGI4ZDgwMWQxMDI4YmYzMWQ0NzM1Y2EwYTlhNTZkZjNiNmM2NjM1YWJmZjRlMGVlYjYwYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M9iFax2VcHuQdylzP7Owl5f7QGiq_mOnJUdk2bHAGDs)
> 
> nice

Hi. Has this been fixed in the latest build? I have the same issue with version v2.9.13_x64.dmg.

**Comment by H0llyW00dzZ** [comment]
> > > I tested the export and import functions using your executable, and everything is working fine now.
> > > ![image](https://private-user-images.githubusercontent.com/36789418/286792026-37c586eb-c776-4fb6-8431-114eb4e30f0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMTA2ODYsIm5iZiI6MTcwMTMxMDM4NiwicGF0aCI6Ii8zNjc4OTQxOC8yODY3OTIwMjYtMzdjNTg2ZWItYzc3Ni00ZmI2LTg0MzEtMTE0ZWI0ZTMwZjBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDAyMTMwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNTkxOGZiOTk1OGI4ZDgwMWQxMDI4YmYzMWQ0NzM1Y2EwYTlhNTZkZjNiNmM2NjM1YWJmZjRlMGVlYjYwYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M9iFax2VcHuQdylzP7Owl5f7QGiq_mOnJUdk2bHAGDs)
> > 
> > 
> > nice
> 
> Hi. Has this been fixed in the latest build? I have the same issue with version v2.9.13_x64.dmg.

nah because it must refactor

**Comment by sheng-di** [comment]
+1，在 macOS 下的客户端也有这个问题。

**Comment by TCOTC** [comment]
fix by #3972

v2.11.3 我这里导出已经正常了，大家可以试一试
